# State Info
Header header

# Position and Orientation
geometry_msgs/PoseStamped pose

#Velocity
geometry_msgs/Twist velocity

#IMUBias
geometry_msgs/Vector3 gyro_bias
geometry_msgs/Vector3 accel_bias


